### PR TITLE
Added github templates copied from jag-file-submission

### PR DESCRIPTION
**Pull request by KDNeufeld**
Added github templates copied from jag-file-submission

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/onboarding-request.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+labels: Onboarding
+assignees: akroon3r, alexjoybc
+
+---
+
+After this issue has been opened, please provide the following information:
+
+- contact email that admins can reach you at
+- a brief description of what you're looking to accomplish by integrating with the efiling-hub
+- a link to github repository of the application that is requesting access to the efiling-hub
```
Comment (karthikmsqa) [inline .github/ISSUE_TEMPLATE/onboarding-request.md:14]: Same as above.

File: .github/ISSUE_TEMPLATE/onboarding-request.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+- a list of the types of documents you're looking to upload to the efiling-hub
```
[the remark this answers is not in the document]
Comment (karthikmsqa) [reply .github/ISSUE_TEMPLATE/onboarding-request.md:15]: In general, Onboarding-request is applicable for Ai reviewer? as it is used for internal processing.

File: .github/ISSUE_TEMPLATE/onboarding-request.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+name: Onboarding Request
+about: Request for access to Efiling Hub API and Frontend
+title: "[ONBOARDING]"
+labels: Onboarding
+assignees: akroon3r, alexjoybc
```
[the remark this answers is not in the document]
Comment (KDNeufeld) [reply .github/ISSUE_TEMPLATE/onboarding-request.md]: see above. removed file

File: .github/ISSUE_TEMPLATE/onboarding-request.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+---
+
+After this issue has been opened, please provide the following information:
+
+- contact email that admins can reach you at
+- a brief description of what you're looking to accomplish by integrating with the efiling-hub
```
[the remark this answers is not in the document]
Comment (KDNeufeld) [reply .github/ISSUE_TEMPLATE/onboarding-request.md]: see above. removed file

File: .github/ISSUE_TEMPLATE/onboarding-request.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (KDNeufeld) [reply .github/ISSUE_TEMPLATE/onboarding-request.md]: see above. removed file

**Review by karthikmsqa — APPROVED** [review]
Looks great, thanks @KDNeufeld

**Pull request merged**